### ISSUE TITLE
Add certificate let`s encrypt

### ISSUE THREAD

**Issue by John-Henrique** [DigitalOceanPHP/DigitalOceanV2#234]
I want add let`s encrypt certificate to domain, but reading class I understand how use method create. Can you help me?

I don't understand, looks like very different of documentation https://developers.digitalocean.com/documentation/v2/#certificates
```
//...
$certificate->create( 'mydomain.com', 'lets_encrypt' );
```

How give domain, type and name?

```

    /**
     * @param string $name
     * @param string $privateKey
     * @param string $leafCertificate
     * @param string $certificateChain
     *
     * @throws HttpException
     *
     * @return CertificateEntity
     */
    public function create($name, $privateKey, $leafCertificate, $certificateChain)
    {
        $data = [
            'name' => $name,
            'private_key' => $privateKey,
            'leaf_certificate' => $leafCertificate,
            'certificate_chain' => $certificateChain,
        ];

        $certificate = $this->adapter->post(sprintf('%s/certificates', $this->endpoint), $data);

        $certificate = json_decode($certificate);

        return new CertificateEntity($certificate->certificate);
    }
```

**Comment by glennjacobs** [comment]
Doesn't look like LetsEncrypt is supported at present. Have flagged as an enhancement.

**Comment by GrahamCampbell** [comment]
~Moving to part of the v4 design issue #249. I am planning to smash out a fresh package for v4 which will feel like the old package, but have none of the issues and all of the compatibility. ;)~

EDIT: Implemented this in v3.0.0.
